src/eei.cpp

```diff
@@ -209,6 +221,8 @@ namespace HeraVM {
       vector<uint8_t> input(msg.input, msg.input + msg.input_size);
       storeMemory(input, dataOffset, resultOffset, length);
 
+      takeGas(GasSchedule::stepGas2 + GasSchedule::copy * (length / 32));
```

Comment:
@axic What do you think about charging for memory per byte, instead of per "word"?
@jakelang I believe `length / 32` should be rounded up, not down.